### PR TITLE
Quill emoji

### PR DESCRIPTION
**Pull request by rbennettcw**
## Link to Issue
Closes: #3699 

## Description of Changes
- Adds FIXME widget to TODO page.
- 

## Test Plan
- Unit tested the `FIXME()` call.
- CA (click around) tested on local and frack:
  - TODO page
  - 

## Deployment Plan

1. 

## Other Considerations

-

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/components/react_quill_editor/react_quill_editor.tsx [hunk 1]
```diff
@@ -94,6 +96,11 @@ const ReactQuillEditor = ({
     setContentDelta,
   });
 
+  // handle emoji selector and shortcuts
+  const { emojiModules } = useEmoji({ editorRef });
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/components/react_quill_editor/react_quill_editor.tsx:100]: kudos @rbennettcw for using all those custom hooks! 💪

**Comment by zakhap** [comment]
Beyond bandwidth, does this PR need anything from Product / Engineering for it to be continued?

**Comment by rbennettcw** [comment]
> Beyond bandwidth, does this PR need anything from Product / Engineering for it to be continued?

Not blocked by product/engineering.

The `quill-emoji` library doesn't work properly. Need to continue debugging it– worst case scenario create a custom quill module which could take a few days since it's a lower-level implementation.

FYI: Mac users can use CTRL+CMD+Space to show the built-in emoji selector and Windows has an equivalent, I believe. Linux would be hit or miss. Just sharing since it's not common knowledge.

**Comment by masvelio** [comment]
@rbennettcw can we either make this PR mergable or close it? It seems emojis are not super priority, so probably could be closed, especially, that PR is not functional.

**Pull request closed without merging**